tests/PHPCurlClass/PHPCurlClassTest.php

```diff
@@ -460,11 +460,13 @@ public function testPatchRequestMethodWithMultidimArray()
             ),
         );
 
+        $encoded = json_encode($data);
```

Comment:
I prefer to have the test variables inherently linked in some manner, so as to prevent future edits to cause them to diverge.  Ideally, `scalar` types (arrays, integers, strings) would be configured in a central configuration file that both the Server and the Test use.  I was looking around here, so I touched it, but I am not going that far yet.